rust/libnewsboat/src/fmtstrformatter/mod.rs

```diff
@@ -119,6 +119,26 @@ impl FmtStrFormatter {
                 result.push_str(text);
                 result.push_str(&padding);
             }
+
+            Padding::Center(total_width) => {
+                let mut w = total_width;
+                if total_width == 0 {
+                    w = _width as usize;
```

Comment:
Underscore means that the variable is not used. Since it *is* used now, please rename it to `width`. (I expect this is something Clippy would warn you about once you've rebased onto current `master` branch).